packages/orbit-components/src/SkipLink/index.js

```diff
@@ -42,31 +42,17 @@ StyledLink.defaultProps = {
   theme: defaultTheme,
 };
 
-const StyledLabel = styled.p`
-  background-color: red;
-  visibility: none;
-  position: absolute;
-  width: 0;
-  height: 0;
-  padding: 0;
-  margin: -1px;
-  overflow: hidden;
-  clip: rect(0, 0, 0, 0);
-  border: 0;
-`;
-
-const SkipLink = ({ links, buttonLabel }: Props) => {
+const SkipLink = ({ links, label }: Props) => {
   return (
-    <StyledNavigation>
-      {buttonLabel && <StyledLabel>{buttonLabel}</StyledLabel>}
+    <StyledNavigation aria-label={label}>
       {links &&
         links.map(({ href, name, onClick }, index) => {
           return (
             <StyledLink
               key={encodeURIComponent(name + index)}
               href={href}
-              tabIndex={href ? "" : "0"}
-              role={href ? "" : "Button"}
+              tabIndex={onClick && "0"}
```

Comment:
Aren't anchor elements focusable by default? What does this do?

Reply:
it's not for anchor, it's for button, when `onClick` is provided, there is change to `role="Button"` and it will not be focusable without `tabIndex="0"`

Reply:
Oh, I see!